tests/items/src/test_fn_attrs.rs

```diff
@@ -57,8 +63,7 @@ pub fn test_fn_attrs() {
 
     if cfg!(not(target_os = "macos")) {
         // aliased_fn is aliased to the inline_extern function
-        assert!(src.contains(
-            "extern \"C\" {\n    #[link_name = \"inline_extern\"]\n    fn aliased_fn();"
-        ));
+        let aliased_fn_syntax = |public| format!("extern \"C\" {{\n    #[link_name = \"inline_extern\"]\n    {}fn aliased_fn();", public);
```

Comment:
Could we use raw string literals here (and below) to make this more readable?

Reply:
Yeah.  Fixed in d4b5e979f.  I used raw strings for all of them, not just the ones touched in this diff.